pkg/cli/objectformats/objectformats.go

```diff
@@ -68,6 +68,31 @@ func GetResourceTableFormat() output.FormatterOptions {
 				Heading:  "TYPE",
 				JSONPath: "{ .Type }",
 			},
+			{
+				Heading:  "STATE",
+				JSONPath: "{ .Properties.ProvisioningState }",
+			},
+		},
+	}
+}
+
+// GetGenericResourceTableFormat() returns a FormatterOptions struct containing three columns, one for the resource name, one for
+// the resource type, and one for the provisioning state.
+func GetGenericResourceTableFormat() output.FormatterOptions {
+	return output.FormatterOptions{
+		Columns: []output.Column{
+			{
+				Heading:  "RESOURCE",
+				JSONPath: "{ .Name }",
+			},
+			{
+				Heading:  "TYPE",
+				JSONPath: "{ .Type }",
+			},
+			{
+				Heading:  "STATE",
+				JSONPath: "{ .Properties.provisioningState }",
+			},
```

Comment:
Isn't this the same as the above? This is all loosely-typed, so it's flexible. We don't need a separate format for each Go type.

Reply:
The difference (and the problem) is that in one of them provisioning state starts with a uppercase letter and in the other one lowercase letter.

Reply:
This isn't great, but I think it's an OK solution for now.

Can you improve the comments of these two functions so that it will explain which one to choose?

Reply:
Done